pkgpanda/__init__.py

```diff
@@ -356,7 +356,7 @@ def load(self, id: str):
         filename = os.path.join(path, "pkginfo.json")
         try:
             pkginfo = load_json(filename)
-        except FileNotFoundError as ex:
+        except OSError as ex:
```

Comment:
@branden , sure, if that happened to be the case here, then, I think, we should catch it earlier.

1) Catching specific exception is `FileNotFoundError` is better than `OSError` because we are looking for specific error.

2) If a `NotADirectoryError` really got raised here, it could be caught in the `self.package_path` or  you could change the statement  `if not os.path.exists(path)` to `if not os.path.isdir(path):`